prudentia/tasks/postfix.yml

```diff
@@ -0,0 +1,40 @@
+---
+  - name: Postfix | Check if is installed
+    command: test -x /usr/sbin/postfix
+    register: postfix_present
+    when: ansible_os_family == "Debian"
+    ignore_errors: yes
+    tags:
+      - postfix
+
+  - name: Postfix | Set up postmail selections
+    command: echo "postfix postfix/main_mailer_type select Internet Site" | debconf-set-selections
+    when: ansible_os_family == "Debian" and postfix_present|failed
+
+    sudo: yes
+    tags:
+      - postfix
+
+  - name: Postfix | Set up postmail selections
+    command: echo "postfix postfix/mailname string {{ansible_fqdn}}" | debconf-set-selections
```

Comment:
I noticed that `ansible_fqdn` is hardly set in new boxes, can we introduce a parameter for the task and document it at the beginning of the task as we usually do?